solr/contrib/ltr/README.md

```diff
@@ -310,8 +310,77 @@ list that will be computed but not used in the model for scoring, which can
 be useful for logging. Params are the LambdaMART specific parameters. In this
 case we have 2 trees, one with 3 leaf nodes and one with 1 leaf node.
 
-A good library for training LambdaMART ( http://sourceforge.net/p/lemur/wiki/RankLib/ ).
-You will need to convert the RankLib model format to the format specified above.
+###### model3.json
```

Comment:
You removed the section talking about training a LambdaMART model.  I think it's a good idea to keep that if someone wants to explore building one.